cypress.json

```diff
@@ -7,5 +7,6 @@
   "requestTimeout": 15000,
   "viewportWidth": 1400,
   "viewportHeight": 660,
-  "projectId": "ftxg4w"
+  "projectId": "ftxg4w",
+  "watchForFileChanges": false
```

Comment:
Doesn't that break dev mode?

Reply:
Shouldn't, it only switch off the automatic test re-run when I save anything in the code. That's a little bit annoying during test creation.